### PR TITLE
Allow set notification area settings

### PR DESCRIPTION
**Pull request by ZacMarcus**
I created a new pull request so the file I added and deleted wouldn't be included

### CODE REVIEW

File: Web/WinConfig.cshtml [hunk 1]
```diff
@@ -75,13 +75,14 @@ Set-WindowsExplorerOptions -DisableShowHiddenFilesFoldersDrives -DisableShowProt
 </pre>
 
 <h3>Set-TaskbarOptions</h3>
-<p>Sets options on the Windows Taskbar</p>
+<p>Sets options on the Windows Taskbar
+AlwaysShowIconsOn/AlwaysShowIconsOff allows turning on or off always show all icons in the notification area</p>
 <pre>
```
Comment (mwrock) [inline Web/WinConfig.cshtml]: Since this is in a single `<p></p>` it will run on one line. We should split it to separate lines.

**Comment by mwrock** [comment]
This is great! Just left a comment about the doc.

**Comment by ZacMarcus** [comment]
Done!

**Comment by mwrock** [comment]
Thanks so much!!

**Pull request merged**